circuits/jwt.circom

```diff
@@ -83,11 +85,11 @@ template JWTVerify(max_msg_bytes, max_json_bytes, n, k) {
     /************************** JWT REGEXES *****************************/
 
     /* ensures signature is type jwt */
-    component type_jwt_regex = MessageType(max_json_bytes);
-    for (var i = 0; i < max_json_bytes; i++) {
-        type_jwt_regex.msg[i] <== message_b64.out[i];
-    }
-    type_jwt_regex.out === 1;
+    // component type_jwt_regex = MessageType(max_json_bytes);
```

Comment:
We could reinstate this and still be under the 1M constraint mark

Reply:
I'm not sure if OpenAI uses the same private key to generate any other tokens/sign anything else. If yes, then we should. If no, then we shouldn't. I think for now let's just keep it removed!